tests/testthat/test-shape-translate.r

```diff
@@ -0,0 +1,81 @@
+context("translate_shape_string")
+
+test_that("strings translate to their corresponding integers", {
+  shape_strings <- c(
+    "square open",
+    "circle open",
+    "triangle open",
+    "plus",
+    "cross",
+    "diamond open",
+    "triangle down open",
+    "square cross",
+    "asterisk",
+    "diamond plus",
+    "circle plus",
+    "star",
+    "square plus",
+    "circle cross",
+    "square triangle",
+    "square",
+    "circle small",
+    "triangle",
+    "diamond",
+    "circle",
+    "bullet",
+    "circle filled",
+    "square filled",
+    "diamond filled",
+    "triangle filled",
+    "triangle down filled"
+  )
+
+  expect_equal(translate_shape_string(shape_strings[1]), 0)
+  expect_equal(translate_shape_string(shape_strings), 0:25)
+
+  expect_equal(translate_shape_string(rep.int(shape_strings[1], 10)),
+               rep.int(0, 10))
+  expect_equal(translate_shape_string(rep(shape_strings[1:3], each = 4)),
+               rep(0:2, each = 4))
+})
+
+test_that("single characters are not translated to integers", {
+  expect_equal(translate_shape_string(letters), letters)
+  expect_equal(translate_shape_string(as.character(0:9)), as.character(0:9))
+})
+
+test_that("non-unique substrings of shape names raise an error", {
+  shape_strings <- c(
```

Comment:
Similarly you only need to test a single string here, and in the next error (and you could probably combine the two tests)